.pr-preview.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should be named as `boilerplate/registry/.pr-preview.json` instead of placed at the top directory?

Reply:
I didn't know that could work. There's an `index.html` in the top directory too, which is for the TTWG homepage. I've added `.pr-preview.json` to the location you suggested as well. But I think PR Preview only does one preview per repo, as far as I know. tobie/pr-preview#18 is open to support multiple specs per repo, but as you can see from the response to my comment, it's awaiting resource.

Reply:
Ah, I thought this `boilerplate/registry` directory was for ready-to-copy respec template to create new repository (e.g. making new repository like [w3c/tt-profile-registry](https://github.com/w3c/tt-profile-registry)). If so, this file should be at the same directory as this template, but not at the top of this repository. OR, If this PR is really to put at the top of this repository with pointing `index.html`, I propose to use `html` for type, instead of `respec`.
As pointed, pr-preview does handle only one file in each repository, yes.

Reply:
That's a good point, having the `.pr-preview.json` file there does no harm, I think, and means there is something ready to copy. I think it's fine to leave it there and keep the type as respec, while understanding that PR Preview won't actually work with it in the current location.